scripts/gee

```diff
@@ -2650,16 +2650,29 @@ function gee__fix() {
         "${BZL_FILES[@]}"
   fi
 
-  # clang-format:
-  local -a CLANG_FILES=()
-  for FNAME in "${FILES[@]}"; do
-    if [[ "${FNAME}" =~ \.h$|\.c$|\.cc$|\.cpp ]]; then
-      CLANG_FILES+=( "${FNAME}" )
+  # clang-format, if available:
+  if [[ -x /usr/bin/clang-format-10 ]]; then
+    local -a CLANG_FILES=()
+    for FNAME in "${FILES[@]}"; do
+      if [[ "${FNAME}" =~ \.h$|\.c$|\.cc$|\.cpp ]]; then
+        CLANG_FILES+=( "${FNAME}" )
+      fi
+    done
+    if (( "${#CLANG_FILES[@]}" )); then
+      _info "clang-format: Fixing ${CLANG_FILES[*]}"
+      _cmd /usr/bin/clang-format-10 --verbose --style=Google -i "${CLANG_FILES[@]}"
```

Comment:
no, we have bazel rules to format according to our style guide (from Google, but with minor corrections)

Reply:
Unfortunately, bazel run :cc_format doesn't take any arguments, and always iterates over all files.  I'll fix that, and switch to //:cc_format.

Reply:
Fixed, done.

Because @clang-format is part of internal's workspace, and clang-format.bzl is part of enkit's repo, it's impossible to test this change without first checking the code in, but it probably works.